setup.py

```diff
@@ -5,6 +5,13 @@
 
 from setuptools import setup, Extension
 
+# Description
+DESCRIPTION = "Non-uniform fast Fourier transforms on the GPU"
+
+with open('python/cufinufft/README.md', encoding='utf8') as fh:
+    README = fh.read().split('\n')
+    LONG_DESCRIPTION = '\n'.join([x for x in README if not x[:3] == '[!['])
```

Comment:
Are you trying to match the string literal `'[!['` ?  I didn't see any matches.

Reply:
Copy paste from another project. We don't need it here so will remove.

Reply:
Gotcha, I thought maybe you wanted a regexp for the links or something.